app/main.py

```diff
@@ -1,8 +1,58 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: List[Car]) -> float:
+        total_cost = 0
+        for car in cars:
+            total_cost += self.wash_single_car(car)
+        return total_cost
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(
+            (
+                car.comfort_class
+                * (self.clean_power - car.clean_mark)
+                * self.average_rating
+            ) / self.distance_from_city_center,
+            1
+        )
+
+    def wash_single_car(self, car: Car) -> float | int:
+        if car.clean_mark < self.clean_power:
+            single_cost = self.calculate_washing_price(car)
+            car.clean_mark = self.clean_power
+            return single_cost
+        return 0
```

Comment:
What if we want only to wash car, without charging and calculating customer? Your `wash_single_car` method should only clean your car, without calculating the price and returning anything. Call both methods inside `serve_cars`